docs: improve consumability and accuracy of fscloud doc (#47)
terraform-ibm-modules/terraform-ibm-icd-postgresql: profiles/fscloud/README.md

# Financial Services Controls - IBM Cloud Databases for ICD Postgresql module

This module is a version of the [parent root module](../../) that comes with pre-defined defaults that meet the relevant controls from the [IBM Cloud Framework for Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about).

See the accompanying [fscloud example](../../examples/fscloud/) leveraging this module.

The defaults coming with this module have been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and the Financial Cloud profile for [Security and Compliance Center](https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-getting-started). They meet all applicable goals.

> :exclamation: The financial services framework mandates the application of an inbound network-based allow-list in front of the ICD PostGresql instance. This can be achieved in multiple ways:

Reviewer: ```suggestion
:exclamation: **Important:** The Financial Services framework mandates the application of an inbound network-based allowlist in front of the ICD PostgreSQL instance. You can comply in multiple ways:
```
